subsys/caf/modules/sensor_sampler.c

```diff
@@ -145,50 +162,48 @@ static void trigger_handler(const struct device *dev, struct sensor_trigger *tri
 {
 	sensor_trigger_set(dev, trigger, NULL);
 
+	const struct sensor_config *sc = get_sensor_config(dev);
```

Comment:
We could assert that sc and sd are not `NULL` (assert could be added to `get_...` functions aswell)

Reply:
ok

Reply:
will add to `get_` funs only, its sufficient.